RiotSwiftUI/Modules/Template/SimpleUserProfileExample/View/TemplateUserProfilePresenceView.swift

```diff
@@ -0,0 +1,66 @@
+// 
+// Copyright 2021 New Vector Ltd
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+import SwiftUI
+
+@available(iOS 14.0, *)
+struct TemplateUserProfilePresenceView: View {
+    
+    // MARK: - Properties
+    
+    // MARK: Public
+    let presense: TemplateUserProfilePresence
+    
+    var body: some View {
+        HStack {
+            Image(systemName: "circle.fill")
+                .resizable()
+                .frame(width: 8, height: 8)
+                .foregroundColor(foregroundColor)
```

Comment:
The foreground colour should be inherited here anyway so this is probably redundant.

Reply:
oh yea, leftover from an edit. Thanks.